ci-operator/step-registry/mco/conf/day2/pintargetrelease/mco-conf-day2-pintargetrelease-ref.yaml

```diff
@@ -0,0 +1,30 @@
+ref:
+  as: mco-conf-day2-pintargetrelease
+  from_image:
+    namespace: ci
+    name: verification-tests
+    tag: latest
+  timeout: 1h
+  grace_period: 10m
+  commands: mco-conf-day2-pintargetrelease-commands.sh
+  cli: latest
+  resources:
+    limits:
+      cpu: "1"
+    requests:
+      cpu: 350m
+      memory: 1Gi
+  env:
+  - name: MCO_CONF_DAY2_PINTARGETRELEASE_MCPS
+    default: "master worker"
+    documentation: |-
+      Space separated list will all the pools that will pin the target image
```

Comment:
@sergiordlr the sentence needs to refine, like `MCP list will be used for creating PinnedImageSet`

Reply:
Done